### PR TITLE
VOTE-668: DOB disallow future date

### PR DESCRIPTION
**Pull request by WilliamYKwon**
[VOTE-668](https://cm-jira.usa.gov/browse/VOTE-668)

Uncommented age verification to verify the user is between 17 and 110.

p.s.
I'm not sure if enabling age verification was in the scope of this ticket, but entering future date was already not allowed as the regex is '200\{1}'

image

## Testing:

1. Go to Personal Information section.
2. In the DOB fields, verify you can't enter dates that make you younger than 16 and older than 120.

### CODE REVIEW

File: src/components/FormSections/PersonalInfo.jsx [hunk 1]
```diff
@@ -37,32 +37,36 @@ function PersonalInfo(props){
...
-            // age <= 110,
-            // age >= 17
+            yearStart <= 20,
+            yearStart >= 19,
+            age <= 120,
+            age >= 16
           ];
 
           if (dobValues.includes(false)) {
+            e.target.setCustomValidity(' ');
```
Comment (SamiraMSadat) [inline src/components/FormSections/PersonalInfo.jsx:69]: @WilliamYKwon Why was this added here? It is not being used as part of the onInput() or onInvalid() functions so I am not sure why the error message is being set to empty.

Comment (WilliamYKwon) [reply src/components/FormSections/PersonalInfo.jsx:69]: From what I understand, onInvalid() doesn't check against checkDateValues() so even if a birthdate of a 13yo is entered, it displays the error but still lets you proceed to the next steps. So I added that there to disallow one from proceeding on.

If this isn't valid to do, I can look into other ways handle this

Comment (SamiraMSadat) [reply src/components/FormSections/PersonalInfo.jsx:69]: @WilliamYKwon Oh okay got it! I just tested again and it works, so this should be good.

**Review by SamiraMSadat — APPROVED** [review]
